app/2.0/docs/devguide/data-binding.md

```diff
@@ -524,6 +524,9 @@ item**.
 <span>{{array.0}}</span>
 ```
 
+**Array binding does not track array mutations.** 
+An array mutation that reorders an array (for example, using the [splice method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice)) will not be reflected in static bindings.
```

Comment:
This doesn't exactly follow, because we just told then that these bindings don't work.

You might say something like,

A binding in the `[[array.0]]` format may _appear_ to work, but it's severely limited:

-   The array index can only be a number literal, not a variable. For example, `[[array.i]]` binds
     to the `i` property of `array`, not `array[i]`. 
-   The binding doesn't update  when the array is mutated. For example, the following would
     update the binding:
    
        this.array[0] = 'New value';
 
     But the following mutation would not:

        // replace first array item
        this.splice('array', 0, 1, 'New value');

That would be correct. But it's a pretty lengthy digression that detracts from the main point of the section, which is "don't do `[[array.0]]`.

Alternately, we could just embed comments in the code snippet.

 ```
 <!-- Don't do this! This format doesn't work -->
 <span>[[array[0]]]</span>
 <!-- Don't do this! Data may display, but won't be updated correctly -->
  <span>[[array.0]]</span>
  ```